client/galaxy/scripts/mvc/tool/tool-form-base.js

```diff
@@ -228,10 +238,28 @@ define(['utils/utils', 'utils/deferred', 'mvc/ui/ui-misc', 'mvc/form/form-view',
                     tooltip : 'Download this tool',
                     onclick : function() {
                         window.location.href = Galaxy.root + 'api/tools/' + options.id + '/download';
-                    }
+                 }
                 });
             }
 
+            // add tours option
+            if (options.tours && options.tours.length > 0) {
+                for (var i in options.tours) {
+                    var tour = options.tours[i];
+                    tours_button.addMenu({
+                        title   : tour.name,
+                        tour    : tour,
+                        tooltip : 'Start an interactive tool introduction',
+                        icon    : 'fa-cube',
+                        onclick : function() {
+                            window.location.href = Galaxy.root + 'api/tools/' + options.id + '/tours/' + tour.tour_id;
```

Comment:
Got it.  Let me read through this and figure out another way.  I'm almost done adding the start_tour_via_url functionality, but I still want to avoid any redirect where possible (like here, I think).